master/buildbot/test/integration/test_worker_comm.py

```diff
@@ -12,10 +12,13 @@
 # Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 #
 # Copyright Buildbot Team Members
+import os
+
 import mock
```

Comment:
not related to your change: `mock` is about testing the functionality so it should be after all non-testing related imports.